### DISCUSSION TITLE
Disable shortcuts in FullpageOS

### DISCUSSION THREAD

**Discussion by Mikkelsen2005** [community/community#120951] [Other Feature Feedback, Questions, & Ideas]
### FullPageOS

How to disable shortcuts?

### Body

I have an issue im trying to build a Raspberry kiosk with FullpageOS.
On the way ive met some issues as of how to disable keyboard shortcuts like CTRL ALT F1 for terminal and such.
Does anyone have a solution for this

**Comment by kakajohang** [comment]
Lik
